src/pages/docs/packaging-applications/package-repositories/artifactory-generic-feeds.md

```diff
@@ -27,6 +27,16 @@ Artifactory generic feeds accommodate files of any type without requiring a spec
 To handle this we depend on the Artifact Path Pattern regex expression available from Artifactory to be set on the feed in Octopus. 
 To find the _Artifact Path Pattern_ go to **Administration ➜ Layouts ➜ Regular Expression View ➜ Resolve** and copy the Artifact Path Regex expression. A new repository will use the default `simple-layout` generated by Artifactory, the regex for this layout is <code>(?<orgPath>.+?)/(?<module>[^/]+)/(?<module>\2)-(?<baseRev>[^/]+?)\.(?<ext>(?:(?!\d))[^\-/]+|7z)</code>.
 
+:::div{.warning}
+Important: To ensure that Octopus can correctly interact with Artifactory generic feeds, the custom layout pattern in Artifactory must include specific naming components. The layout pattern must specify the artifact name format as `[module]-[baseRev].[ext]`. It's crucial that the `[module]` and `[baseRev]` components are present, and that `[baseRev]` conforms to Semantic Versioning.
```

Comment:
I don't think the artifact name format is a requirement. We only rely on the regex return match groups named [baseRev](https://github.com/OctopusDeploy/OctopusDeploy/blob/main/source/Octopus.Core/Packages/ArtifactoryGeneric/ArtifactoryGenericPackageFeed.cs#L112), [module](https://github.com/OctopusDeploy/OctopusDeploy/blob/main/source/Octopus.Core/Packages/ArtifactoryGeneric/ArtifactoryGenericPackageFeed.cs#L134) and [ext](https://github.com/OctopusDeploy/OctopusDeploy/blob/main/source/Octopus.Core/Packages/ArtifactoryGeneric/ArtifactoryGenericFeedClient.cs#L117)